Raspi/Test/mqtt_wireless_log.py

```diff
@@ -154,7 +169,8 @@ def merge_temps(output_filename, temp_filepaths):
 
     # Merge the dataframes and output the final csv
     merged_dataframe = pd.concat(temp_dataframes, axis=1)
-    merged_dataframe.to_csv(output_filename)
+    file_path = os.path.dirname(__file__)
```

Comment:
I haven't looked into the cause, but it seems that when you run `python mqtt_wireless_log.py` from inside the `Test` directory, it fails to save the files because `os.path.dirname(__file__)` is for some reason returning an empty string, causing python to panic and stop everything.

It works fine if you run the script with `python Test/mqtt_wireless_log.py` from the `Raspi` directory.